packages/imperative/src/cmd/src/syntax/__tests__/SyntaxValidator.unit.test.ts

```diff
@@ -525,6 +525,19 @@ describe("Imperative should provide advanced syntax validation rules", () => {
             expect(svResponse.valid).toEqual(true);
         });
 
+        it.only("If a positional argument does not match the defined regex, the command should fail", async () => {
```

Comment:
Please allow for other tests to run on this test suite 😋 
```suggestion
        it("should fail if a positional argument does not match the defined regex", async () => {
```